plugins/woocommerce/includes/admin/views/html-admin-dashboard-setup.php

```diff
@@ -27,3 +27,29 @@
 	</div>
 	<div class="clear"></div>
 </div>
+
+<script type="text/javascript">
+	/*global jQuery */
+	(function( $ ) {
+		const widget = $( '.dashboard-widget-finish-setup' );
+		const currentStep = widget.data( 'current-step' );
+		const totalSteps = widget.data( 'total-steps' );
+		console.log(currentStep);
+		console.log(totalSteps);
```

Comment:
I think that we should remove the `console.log`s.

Reply:
Great catch!  Don't want those in there 😵‍💫